plugins/home/README.md

```diff
@@ -43,9 +43,14 @@ import { HomepageCompositionRoot } from '@backstage/plugin-home';
 import { HomePage } from './components/home/HomePage';
 
 // ...
-<Route path="/" element={<HomepageCompositionRoot />}>
-  <HomePage />
-</Route>;
+<Route
```

Comment:
Hey @Fox32, versions looks good to me. But looking at the changes of the docs I'm curious in what does not work here? Looking at the [App](https://github.com/backstage/backstage/blob/master/packages/app/src/App.tsx#L117) the example in the readme is equal to how the element prop and children is used there.

Reply:
I was also trying setting up it myself now using the docs and adding the following to the App.

```
<Route path="/" element={<HomepageCompositionRoot />}>
  <HomePage />
</Route>
```

which worked well. But happy to make these changes if required, I just want to make sure there is nothing else which is the root cause to why it doesn't work as expected!

Reply:
You are right, it should actually work and it works for other plugins. But once I do it in our app, I miss all the widgets from the homepage:
![image](https://user-images.githubusercontent.com/648527/133084574-a590129a-8f77-4d86-b9fd-db4e3499196a.png)

Would be fine to keep it as is for now and just fix it locally on our side. But I would still remove the semicolon 😉

Reply:
Arg, pettier always includes the semicolon, anyway let's keep it as-is.

Reply:
interesting, do you now why it doesn't give you the widgets?

Reply:
No idea, we use the same pattern for the scaffolder, explore and search, no problems there.

Reply:
Alright, the only thing I can think of is looking into your HomePage component in the app which should live under `packages/app/src/components/home/HomePage.tsx` if I look at [the docs](https://github.com/backstage/backstage/blob/master/plugins/home/README.md#setting-up-the-home-page), what that looks like and if its missing the widgets!